module/zfs/zio_crypt.c

```diff
@@ -369,7 +369,7 @@ zio_crypt_key_get_salt(zio_crypt_key_t *key, uint8_t *salt)
 /*
  * This function handles all encryption and decryption in zfs. When
  * encrypting it expects puio to reference the plaintext and cuio to
- * reference the cphertext. cuio must have enough space for the
+ * reference the cyphertext. cuio must have enough space for the
```

Comment:
The correct (en_US) spelling is ciphertext.

Reply:
Thanks! For Wikipedia they are both valid:
«ciphertext or cyphertext is the result of encryption performed on plaintext»

You tell me if we can leave it alone or change it.

Reply:
"Thanks" was for the review. Not ironic.

Reply:
I grepped. There are no other instances of "cypher". Please use "ciphertext".

Reply:
Done!